xbmc/windowing/windows/WinSystemWin32.cpp

```diff
@@ -444,6 +444,9 @@ bool CWinSystemWin32::ChangeResolution(RESOLUTION_INFO res)
 {
   const MONITOR_DETAILS &details = GetMonitor(res.iScreen);
 
+  if (!&details)
```

Comment:
It can be wrong, GetMonitor returns m_MonitorsInfo[0] while m_MonitorsInfo is empty.